recipe/post-link.sh.in

```diff
@@ -2,13 +2,37 @@
 for file in \
 	bin/grass \
 	opt/grass/etc/python/grass/app/resource_paths.py \
-	opt/grass/etc/python/grass/lib/*.py \
 	opt/grass/etc/fontcap \
 	opt/grass/demolocation/.grassrc85 \
-	opt/grass/docs/html/*.html \
 	; do
-	sed -Ei \
-		-e "s#@compile_prefix@#$PREFIX#g" \
-		-e "s#lib64/grass85#opt/grass#g" \
-		$PREFIX/$file
+	if [ -f "$PREFIX/$file" ]; then
```

Comment:
Is it just for safety or did you have any issues without this check? I don't see any reason for separating wildcards from this for loop. We know those directories exist and I don't see again a reason for an extra check for directory existence.